src/System.Data.Common/tests/System/Data/DataColumnExpressionTest.cs

```diff
@@ -111,21 +111,21 @@ public static IEnumerable<object[]> Aggregations()
                 foreach (var aggregation in aggregations)
                 {
                     var resultType = type;
-
+                    //TODO
                     // BUG? sum(Column) is always promoted to SqlInt64
```

Comment:
what are these `BUG?` comments? is it something we should look into?

Reply:
I'm not on the System.Data team, but I believe most of these tests were ported from .NET Framework tests written many years ago, and some of them have commentary like this because the feature was being developed at the same time. It's not necessarily useful commentary anymore.